src/TodoList.jsx

```diff
@@ -18,9 +20,7 @@ const TodoList = () => {
         <ul>
             {todoList.map(item => {
                 return (
-                    <li key={item.id}>
-                        {item.title}
-                    </li>
+                    <TodoListItem key={item.id} todo={item.title}/>
```

Comment:
To be consistent with the naming, pass the entire item as the `todo` prop instead of just the title.

Reply:
This will allow for flexibility down the line if you add new data to each todo object (e.g., a due date).